<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShutdownPrompt</class>
 <widget class="QDialog" name="ShutdownPrompt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="iconLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Do you want PonyEdit to remember which files you had open next time?</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="remember">
      <property name="text">
       <string>Remember</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="dontRemember">
      <property name="text">
       <string>Don't Remember</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="dontPrompt">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Don't ask me this again</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
